### ISSUE TITLE
METS/MODS Darmstadt

### ISSUE THREAD

**Issue by igorbajena** [slub/dlf-3d-viewers#90]
I received a sample of METS/MODS file from Darmstadt. I try to test it with the DFG 3D-Viewer by putting the link withtin DFG 3D-Viewer URL, but it didn't work:

METS/MODS by Darmstadt: https://ingest.nfdi4ing.de/api/v1/oai-pmh?verb=ListRecords&metadataPrefix=mets

Test with DFG 3D-Viewer: https://3dtest.dfg-viewer.de/viewer?tx_dlf[id]=https://ingest.nfdi4ing.de/api/v1/oai-pmh?verb=ListRecords&metadataPrefix=mets&no_cache=1  
  
Can you look at it, what is wrong? It could be that the reason is that link is not finished with ".xml" extension?

**Comment by ClemensBeck** [comment]
@igorbajena can you please test if everything is working with the DFG 3D-Viewer Test System (see instructions here: https://github.com/slub/dlf-3d-viewers/blob/main/README.md#dfg-3d-viewer-test-system)? If you have any questions, Markus or I are at your disposal.

**Comment by igorbajena** [comment]
I've tried to create a test system on my computer, and here is the list of issues which I had:
1. How to clone the git -> Probably for coding people it's obvious, for me it was not. I found github instructions, where I can find a link to copy (https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository), but I still had no idea what to do with it. I found some tutorials on YouTube on how to do it, and after overcoming some obstacles, I managed to do it, but I felt that I just didn't have enough skills to do it from the very first instruction point. If we won't not IT audience to use it, it would be nice to add references where they could find more detailed instruction
2. How to run bash script -> That was also not obvious to me;.I don't have expertise in working with a terminal at all. Having the command which should be used would be nice. 
3. After running the script, I got notification that ZIP files are missing:

install.sh: line 17: wget: command not found
unzip:  cannot find or open /d/Work Files/00_AI MAINZ/03_DE_3D-DFG-VIEWER/Stage2/DFG-3D-Viewer_Test_Instance/dlf-3d-viewers/dlf-3dhop/modules/module.zip, /d/Work Files/00_AI MAINZ/03_DE_3D-DFG-VIEWER/Stage2/DFG-3D-Viewer_Test_Instance/dlf-3d-viewers/dlf-3dhop/modules/module.zip.zip or /d/Work Files/00_AI MAINZ/03_DE_3D-DFG-VIEWER/Stage2/DFG-3D-Viewer_Test_Instance/dlf-3d-viewers/dlf-3dhop/modules/module.zip.ZIP.
rm: cannot remove '/d/Work Files/00_AI MAINZ/03_DE_3D-DFG-VIEWER/Stage2/DFG-3D-Viewer_Test_Instance/dlf-3d-viewers/dlf-3dhop/modules/module.zip': No such file or directory

It was basically for every viewer, so I am not sure if it was successful or not. I guess I would need a meeting with you two to help me get through this process. I think it would be also beneficial for me to understand it step by step integration process.

I remembered that Markus also prepared this instance where DFG 3D-Viewer can run without installation, but I couldn't run the action. Probably, I don't have proper permissions.

**Comment by ClemensBeck** [comment]
Dear @igorbajena thank you very much, but you've really made it too complicated for yourself. You should not drag the test system onto your computer, but use the test system that MArkus has set up. You can do this simply by following the instructions under the [link](https://github.com/slub/dlf-3d-viewers/blob/main/README.md#dfg-3d-viewer-test-system). I've just had a look and you should actually have the necessary authorisations, do you really not see a ‘Run Workflow’ button when you call up this [link](https://github.com/slub/dlf-3d-viewers/actions/workflows/run-testsystem.yml)

**Comment by igorbajena** [comment]
You're right, I misunderstood the assignment at first. And yes, I do not see 'Run Worfklow' button under the link you shared (see screenshot below). 
![image](https://github.com/user-attachments/assets/5fb670a1-1899-438b-b972-acc2a35edb0f)

**Comment by ClemensBeck** [comment]
@igorbajena I'm glad to hear that! Have you been able to test whether it works or could you please do so?

**Comment by igorbajena** [comment]
I meant the I do NOT have this "Run Workflow" button, so I was **not able to** do any test
![image](https://github.com/user-attachments/assets/f2900433-ebad-4c7e-a440-01444e42bb51)

**Comment by markusweigelt** [comment]
@igorbajena Sorry you had no write access to the repo now it should work.

**Comment by igorbajena** [comment]
@markusweigelt Thank you, now I was able to run the system.
@ClemensBeck the METS/MODS from Darmstadt doesn't work with test system. But, as Markus mentioned on one of the previous meetings, we should first prepare standard for metadata required by us for validation of provided information. I was postponing it on my agenda, but I'll try to prepare sth for Wednesday. 

@markusweigelt and @MarcMoschSLUB -> Do you maybe have any example of metadata requirements for DFG Viewer for other media, like images or videos? It would be helpful to have a common base on which I would add 3D-oriented metadata.

**Comment by markusweigelt** [comment]
@igorbajena Great!

Here is a METS with images https://digital.slub-dresden.de/data/kitodo/TheDarea_416971482-19100223/TheDarea_416971482-19100223_mets.xml. Afaik audios, videos are in development and not supported atm.

**Comment by igorbajena** [comment]
@markusweigelt Thanks! It should be enough for now
